spec/registry.xml

```diff
@@ -243,11 +243,72 @@
       </option>
     </formatSignature>
 
+    <matchSignature>
+      <input validationRule="anyNumber"/>
+      <option name="select" values="plural ordinal exact" default="plural">
+        <description>
+          The number selection type.
+
+          With `exact` selection, only a variant with a key matching
+          the JSON representation of the input number (without an exponent) will be selected.
+          With `plural` and `ordinal` selection, if no such exactly matching key is present,
+          a variant with the matching CLDR plural category (`zero`/`one`/`two`/`few`/`many`/`other`)
+          for the corresponding selection type will be selected,
+          according to the rules of the current locale
+        </description>
+      </option>
+      <option name="minimumIntegerDigits" values="positiveInteger" default="1">
+        <description>
+          The minimum number of integer digits to use.
+          A value with a smaller number of integer digits than this number will be
+          left-padded with zeros (to the specified length) when formatted.
+        </description>
+      </option>
+      <option name="minimumFractionDigits" values="positiveInteger">
+        <description>
+          The minimum number of fraction digits to use.
+          The default for plain number and percent formatting is 0;
+          the default for currency formatting is the number of minor unit digits provided by
+          the ISO 4217 currency code list (2 if the list doesn't provide that information).
+        </description>
+      </option>
+      <option name="maximumFractionDigits" values="positiveInteger">
+        <description>
+          The maximum number of fraction digits to use.
+          The default for plain number formatting is the larger of minimumFractionDigits and 3;
+          the default for currency formatting is the larger of minimumFractionDigits and the number of minor
+          unit digits provided by the ISO 4217 currency code list (2 if the list doesn't provide that information);
+          the default for percent formatting is the larger of minimumFractionDigits and 0.
+        </description>
+      </option>
+      <option name="minimumSignificantDigits" values="positiveInteger" default="1">
+        <description>
+          The minimum number of significant digits to use.
+        </description>
+      </option>
+      <option name="maximumSignificantDigits" values="positiveInteger" default="21">
+        <description>
+          The maximum number of significant digits to use.
+        </description>
+      </option>
+      <match validationRule="anyNumber" values="zero one two few many other" />
+    </matchSignature>
```

Comment:
Are we missing some options? I'd expect to see a control for "groupingUsed" and "numberingSystem" and possibly some of the other common options (rounding, etc.)

Reply:
The grouping & numbering systems don't affect the plural categorization, and the rounding options are not included for formatting either.

The options added here are the subset of currently defined formatting options that affect plural category selection.